src/p11_hw_feature_type.ml

```diff
@@ -1,2 +1,3 @@
-type t = Pkcs11.CK_HW_FEATURE_TYPE.t
+type t = Pkcs11_CK_ULONG.t
```

Comment:
Right. I think that a `P11_ulong` module will be handy for that but not in this PR.